packages/api/src/controllers/asset.ts

```diff
@@ -169,6 +169,30 @@ async function getActiveObjectStore(id: string) {
   return os;
 }
 
+export type StaticPlaybackInfo = {
+  playbackUrl: string;
+  size: number;
+  height: number;
+  width: number;
+  bitrate: number;
+};
+
+export function getStaticPlaybackInfo(
+  {}: Request["config"],
+  asset: WithID<Asset>,
+  os: ObjectStore
+): StaticPlaybackInfo[] {
+  return (asset.files ?? [])
+    .filter((f) => f.type === "static_transcoded_mp4")
+    .map((f) => ({
+      playbackUrl: pathJoin(os.publicUrl, asset.playbackId, f.path),
+      size: f.spec?.size ?? null,
+      height: f.spec?.height ?? null,
+      width: f.spec?.width ?? null,
+      bitrate: f.spec?.bitrate ?? null,
+    }));
```

Comment:
WDYT of sorting these for a consistent response? For descending bitrate order:
```suggestion
    })).sort((a, b) => b.bitrate - a.bitrate);
```

(apparently `null == 0` for this, no need even to coalesce)

Reply:
Cool :)